packages/react-menu/src/components/Menu/index.ts

```diff
@@ -1,5 +1,13 @@
 export * from './Menu';
-export * from './Menu.types';
+// Explicit exports to ommit MenuCommon
+export type {
+  MenuSlots,
+  MenuProps,
+  MenuState,
+  MenuOpenChangeData,
+  MenuContextValues,
+  MenuOpenEvents,
+} from './Menu.types';
```

Comment:
nit: Order these alphabetically
```suggestion
export type {
  MenuContextValues,
  MenuOpenChangeData,
  MenuOpenEvents,
  MenuProps,
  MenuSlots,
  MenuState,
} from './Menu.types';
```